src/main/java/org/apache/mesos/specification/TaskTypeSpecification.java

```diff
@@ -0,0 +1,8 @@
+package org.apache.mesos.specification;
+
+/**
+ * A TaskTypeSpecification describes the count of a particular TaskSpecification which should be launched.
```

Comment:
It isn't just a count.  It's also the name and the command and the resources and in the future volume specifications etc.